conf/modules.config

```diff
@@ -91,32 +91,32 @@ process {
             enabled: params.bgc_savefilteredcontigs,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
-        ext.args = [
+        ext.args   = [
             "--min-len ${params.bgc_mincontiglength}"
         ].join(' ').trim()
     }
 
     withName: PROKKA {
-        ext.prefix =  { "${meta.id}_prokka" } // to prevent pigz symlink problems of input files if already uncompressed during post-annotation gzipping
```

Comment:
I wouldn't mind leaving comments like this as reminders, but no strong opinion.

Reply:
These are from the new formatter that does partial harshil alignment, unforutnaltey they previously removed :(

i _think_ this was fixed, I will try to re-add them

Reply:
Still not fixed here sadly :(